lib/module-resolvers/file-resolver.js

```diff
@@ -99,9 +126,40 @@ class FileResolver {
         return this._read_file(filename);
     }
 
+    _search_adapter_import(module_path, module_name, importer_path) {
+
+        // Remove the top 'adapters' directory
+        let parts = module_path.split(path.sep).slice(1);
```

Comment:
Hmm, this makes Juttles incompatible between Unix and Windows because one would have to use `\` as a separator on Windows. I guess we just need to hard-code `/` here.